src/Bridge/Symfony/Routing/CachedRouteNameResolver.php

```diff
@@ -51,7 +51,7 @@ public function getRouteName(string $resourceClass, $operationType): string
             // do nothing
         }
 
-        $routeName = $this->decorated->getRouteName($resourceClass, $operationType);
+        $routeName = $this->decorated->getRouteName($resourceClass, $operationType, @func_get_arg(2));
```

Comment:
Can you do it like that instead https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Serializer/Serializer.php#L182-L190